utils/eventtracking.js

```diff
@@ -109,7 +122,58 @@ const getEventTrackingConsent = async () => {
     }
 };
 
-const askForConsentIfNeeded = async (options) => {
+const getIdTrackingConsent = async () => {
```

Comment:
I think we should only ask people one question, i.e. it's enough to just add text in question above in `getEventTrackingConsent`

Reply:
I do not think so, one question is to generate session id and another one is to track account id which is not the same. @vgrichina